src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -384,6 +388,7 @@ public Page Pop()
 
                     //Invoke Page events
                     curTop.InvokeDisappeared();
+                    NotifyAccessibilityStatesChangeOfPages(curTop, null);
```

Comment:
I think that this line needs to be removed.
Instead, the following code is required at line 407.
```
NotifyAccessibilityStatesChangeOfPages(curTop, newTop);
```

Reply:
done